src/pudl/metadata/fields.py

```diff
@@ -998,6 +1018,14 @@
         "description": "Code identifying a dataset available within PUDL.",
         "constraints": {"enum": list(SOURCES)},
     },
+    "date_filed": {
+        "type": "datetime",
+        "description": "Date filing was submitted.",
+    },
```

Comment:
Do we already have a field name that means this, maybe in the context of the FERC XBRL data, which could be re-used here rather than creating a new field name?

Reply:
I wasn't able to find any fields like that, but I can definitely change this to use an existing field if there's one that makes sense.